org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/compiler/AbstractXbaseCompiler.java

```diff
@@ -569,7 +569,13 @@ protected String getFavoriteVariableName(EObject ex) {
 			return ((JvmIdentifiableElement) ex).getSimpleName();
 		}
 		if (ex instanceof XAbstractFeatureCall) {
-			String name = nameProvider.getSimpleName(((XAbstractFeatureCall) ex).getFeature());
+			JvmIdentifiableElement feature = ((XAbstractFeatureCall) ex).getFeature();
```

Comment:
this is part of the separate pr?

Reply:
Prerequisites are merged now. The changes don't appear anymore after the rebase.